rareapi/models/RareUsers.py

```diff
@@ -21,3 +21,13 @@ def username(self):
     @property
     def is_staff(self):
         return self.user.is_staff
+
+    @property
+    def authors_following(self):
+        """Property to easily access list of authors as RareUsers that user is currently subscribed to"""
+        # get QuerySet of subscription objects where this RareUser is the `follower` that are active
+        # (active subscription == subscription where ended_on is None)
+        active_subscriptions = self.follows_subscriptions.filter(ended_on__isnull=True)
```

Comment:
This is super cool! The way I understand this is that django is adding a follows_subscriptions property onto the RareUser object, because the subscriptions model has a foreign key to the Rare user with the related name follows_subscriptions.